.travis.yml

```diff
@@ -27,11 +35,22 @@ script:
       && [ "${TRAVIS_PULL_REQUEST_BRANCH}" != "" ] \
       && [ "${GITHUB_TOKEN}" != "" ];
   then
-    echo "⚙️ 😄 ⚙️"
-    echo "⚙️ 😄 ⚙️ COMING SOON: This build qualifies for staging and commenting"
-    echo "⚙️ 😄 ⚙️"
+    gcloud config set project web-confluence || exit $?
+    gcloud auth activate-service-account --key-file client-secret.json || exit $?
+    gcloud app deploy -q --no-promote --version=${TRAVIS_PULL_REQUEST_BRANCH} . || exit $?
+    DEPLOYED_URL=$(gcloud app versions describe ${TRAVIS_PULL_REQUEST_BRANCH} -s default | grep -Po 'versionUrl: \K.*$')
+    echo "Deployed to ${DEPLOYED_URL}"
+    POST_BODY='{"body": "Staging instance deployed by Travis CI!\nRunning at '"${DEPLOYED_URL}"'"}'
+    curl -H "Authorization: token ${GITHUB_TOKEN}" \
```

Comment:
Might want to include a SHA in the body or similar; this will run every time the PR is updated (lots of comments)